tests/test_overscanCorrection.py

```diff
@@ -268,6 +305,115 @@ def notest_SplineOverscanCorrection(self):
+        inputExp = isrMock.RawMock().run()
+
+        amp = inputExp.getDetector()[0]
+        ampI = inputExp.maskedImage[amp.getRawDataBBox()]
+        overscanI = inputExp.maskedImage[amp.getRawHorizontalOverscanBBox()]
+
+        for fitType in ('MEAN', 'MEDIAN', 'MEDIAN_PER_ROW', 'MEANCLIP', 'POLY', 'CHEB',
+                        'NATURAL_SPLINE', 'CUBIC_SPLINE', 'UNKNOWN'):
+            if fitType in ('NATURAL_SPLINE', 'CUBIC_SPLINE'):
+                order = 3
+            else:
+                order = 1
+
+            if fitType == 'UNKNOWN':
+                with self.assertRaises(ValueError,
+                                       msg=f"overscanCorrection overscanIsNotInt fitType: {fitType}"):
+                    ipIsr.overscanCorrection(ampI, overscanI, fitType=fitType,
+                                             order=order, collapseRej=3.0,
+                                             statControl=None, overscanIsInt=False)
+            else:
+                response = ipIsr.overscanCorrection(ampI, overscanI, fitType=fitType,
+                                                    order=order, collapseRej=3.0,
+                                                    statControl=None, overscanIsInt=False)
+            self.assertIsInstance(response, pipeBase.Struct,
+                                  msg=f"overscanCorrection overscanIsNotInt Bad response: {fitType}")
+            self.assertIsNotNone(response.imageFit,
+                                 msg=f"overscanCorrection overscanIsNotInt Bad imageFit: {fitType}")
+            self.assertIsNotNone(response.overscanFit,
+                                 msg=f"overscanCorrection overscanIsNotInt Bad overscanFit: {fitType}")
+            self.assertIsInstance(response.overscanImage, afwImage.MaskedImageF,
+                                  msg=f"overscanCorrection overscanIsNotInt Bad overscanImage: {fitType}")
+
+    def notest_overscanCorrectionMedianPerRow(self):
+        """Expect that this should reduce the image variance with a full fit.
+        fitType of MEDIAN_PER_ROW will reduce the median value.
+
+        This needs to operate on a RawMock() to have overscan data to use.
+
+        The output types may be different when fitType != MEDIAN_PER_ROW.
+        """
+        self.config.overscan.fitType = 'MEDIAN_PER_ROW'
+        statBefore = computeImageMedianAndStd(self.inputExp.image[self.amp.getRawDataBBox()])
+        oscanResults = self.task.overscanCorrection(self.inputExp, self.amp)
+        self.assertIsInstance(oscanResults, Struct)
+        self.assertIsInstance(oscanResults.imageFit, afwImage.ImageF)
+        self.assertIsInstance(oscanResults.overscanFit, afwImage.ImageF)
+        self.assertIsInstance(oscanResults.overscanImage, afwImage.MaskedImageF)
+
+        statAfter = computeImageMedianAndStd(self.inputExp.image[self.amp.getRawDataBBox()])
+        self.assertLess(statAfter[0], statBefore[0])
+
+    def notest_overscanCorrection_isInt(self):
```

Comment:
These tests are still disabled.

Reply:
Coverage now provided by other tests and this method removed.